src/libraries/System.Text.Json/tests/Serialization/NumberHandlingTests.cs

```diff
@@ -599,6 +599,7 @@ private static void AssertDictionaryElements_StringValues(string serialized)
             }
         }
 
+        [SkipOnCoreClr("Long running test on Checked mode", RuntimeConfiguration.Checked)]
```

Comment:
nit: I usually see attributes that are related to skipping tests as one of the last before the method definition, so here we'd have:

```cs
[Fact]
[ActiveIssue("https://github.com/dotnet/runtime/issues/39674", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))]
[SkipOnCoreClr("Long running test on Checked mode", RuntimeConfiguration.Checked)]
public static void DictionariesRoundTrip()
{
...
```

Applies here and elsewhere